server/cmd/mmctl/commands/post.go

```diff
@@ -41,6 +41,16 @@ var PostListCmd = &cobra.Command{
 	RunE: withClient(postListCmdF),
 }
 
+var PostDeleteCmd = &cobra.Command{
+	Use:   "delete [posts]",
+	Short: "Permanently delete a post",
+	Long: `Permanently delete some posts.
+Permanently deletes one or multiple posts along with all related information including files from the database.`,
+	Example: "  post delete postId1",
```

Comment:
I wonder if we should use a "real" post ID to demonstrate what kind of data the user should provide here.